docker/main/rootfs/etc/s6-overlay/s6-rc.d/chroma/run

```diff
@@ -0,0 +1,16 @@
+#!/command/with-contenv bash
+# shellcheck shell=bash
+# Start the Frigate service
+
+set -o errexit -o nounset -o pipefail
+
+# Logs should be sent to stdout so that s6 can collect them
+
+# Tell S6-Overlay not to restart this service
+s6-svc -O .
+
+echo "[INFO] Starting ChromaDB..."
```

Comment:
I would need to double check S6 but I think it would be preferable to have this service check the config and only run if it is enabled by the user.

Reply:
Good thought on this service. I can read the config file in this bash script and not start the below service if `semantic_search` is not enabled.

Reply:
I looked into this, and it's not easy to do without either dynamically linking files into s6-rc.d at runtime or adding a `down` file and removing it at runtime, which requires a writable rootfs. This service is pretty lightweight in and of itself, and won't really be doing anything unless a client connects to it and makes calls...